src/devices/Ssd1351/Command/Commands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.Ssd1351.Command
+{
+    public enum SSD1351Command : byte
+    {
+        SSD1351_CMD_SETCOLUMN = 0x15, // used
```

Comment:
unsure what `//used` refers to but presumably you might want to remove the comments